subsys/net/lib/lwm2m/lwm2m_shell.c

```diff
@@ -54,34 +53,27 @@ static int cmd_send(const struct shell *sh, size_t argc, char **argv)
 	int ret = 0;
 	struct lwm2m_ctx *ctx = lwm2m_rd_client_ctx();
 	int path_cnt = argc - 1;
-	bool confirmable = true;
 	int ignore_cnt = 1; /* Subcmd + arguments preceding the path list */
 
 	if (!ctx) {
 		shell_error(sh, "no lwm2m context yet\n");
 		return -ENOEXEC;
 	}
 
-	if (argc < 2) {
+	if (argc < 1) {
```

Comment:
Actually, this is wrong.
argc is always at least one. The `argv[0]` is the `"send"` and first arquement is going to be in `argv[1]`

So don't change this line.

Also, now that there is no optional parameters, you can drop the `ignore_cnt` variable. Previously it was used to skip `-n` from parameter list and assume that all the remaining parameters are just paths. Now we assume that all that follows are paths.